server/controllers/utils.js

```diff
@@ -41,11 +44,11 @@ function calculateGradeLadder(grade) {
     rankings.sort((t1, t2) => t2.keyStats.totalPoints - t1.keyStats.totalPoints)
 
     // assign ranks (teams with same points get the same rank)
-    if(rankings.length > 0) {
+    if (rankings.length > 0) {
```

Comment:
Sorry about this, don't know how to configure my code formatter to be happy with no space between if statements :(